src/Shortcodes/FileShortcodeProvider.php

```diff
@@ -136,17 +136,30 @@ public static function handle_shortcode($arguments, $content, $parser, $shortcod
      */
     protected static function getCachedMarkup($cache, $cacheKey, $arguments): string
     {
+        // Retrieve the cache markup
         $item = $cache->get($cacheKey);
+        if (empty($item) || empty($item['markup'])) {
+            // This is a cache miss
+            return '';
+        }
+
+        // Check the file exists
+        $filename = $item['filename'];
+        $hash = $item['hash'];
```

Comment:
We checked whether the `$item['markup']` index was empty above, but not these? Why do we need to check markup but we don't need to check these two? The original implementation did check if the `$item['filename']` index was empty.

Note that `empty($item['filename'])` is different from `$filename = $item['filename']; empty($filename)`

Reply:
Switched to using the coalescing operator.